### PR TITLE
[ADD] Base Phone Rate

### LINKED ISSUE #33: [ADD] base_phone_rate
This module allows you to store the International phone rate of your carrier.

# Usage

* Go to Contacts > Configuration > Localizations > Phone Rates
* Add Phone Rates by entering the name, country, state, dial prefix and rate

# Technical Specifications

## decimal.precision

- [ ] Add a record "Phone Rate" with 5 digits

## phone.rate

- [ ] Add a required char field "Name" (name)
- [ ] Add a required many2one field "Country" (country_id)
- [ ] Add a many2one field "State" (state_id)
- [ ] Add a required char field "Dial Prefix" (dial_prefix). Add a uniqueness constraint.
- [ ] Add a required float field "Rate" (rate)

### PR DESCRIPTION
**Pull request by osi-scampbell**
The following PR adds base_phone_rate to the vertical-isp repo.

Closes #33

### CODE REVIEW

**Review by kvedder-amplex — APPROVED** [review]
Did a demo in a VM with @osi-scampbell. This module looks to work well and I expect it to work in our own instances.  I don't see a need to run this code in house before approval. Thanks for the good work @osi-scampbell

File: base_phone_rate/models/phone_rate.py
Comment (dreispt) [inline base_phone_rate/models/phone_rate.py:9]: It is good practice for the first word to refer to the App or domain the Model belongs to. I suggest:
```suggestion
    _name = "isp.phone.rate"
```

File: base_phone_rate/security/ir.model.access.csv
Comment (dreispt) [inline base_phone_rate/security/ir.model.access.csv]: Regular users shouldn't have write access on this.
```suggestion
access_phone_rate_user,phone.rate.user,model_phone_rate,base.group_system,1,1,1,1
```

Now, we need a master `vertical_isp` App module creating the basic Groups, "ISP Manager" and "ISP User", and then revisit this.

File: base_phone_rate/models/phone_rate.py
Comment (dreispt) [inline base_phone_rate/models/phone_rate.py:16]: Idea for future/roadmap (would be a change request): 
I wonder if we should also include a "Product" here.
We might want to be able to set also for SMS, MMS, etc.

**Review by dreispt — DISMISSED** [review]
LGTM (Runbot was not available for UI tests)

**Review by dreispt — CHANGES_REQUESTED** [review]
LGTM, but I left a few changes to consider.

**Pull request merged**
